src/libraries/Common/src/Interop/Windows/Ole32/Interop.IStream.cs

```diff
@@ -62,5 +63,55 @@ internal interface IStream
 
             IStream Clone();
         }
+
+        // This needs to be duplicated from IStream above because ComWrappers doesn't support re-using
+        // ComImport interfaces. The above interface can be deleted once all usages of the built-in COM support
+        // are converted to use ComWrappers.
+        internal interface IStreamComWrapper
+        {
+            static readonly Guid Guid = new Guid(0x0000000C, 0x0000, 0x0000, 0xC0, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x46);
```

Comment:
If you wanted to be COM centric you could name this member `IID`. That would be my preference because `Guid` is the type not what it is.